build.sbt

```diff
@@ -150,7 +150,10 @@ inThisBuild(
         .map { configKey => s"-D$configKey=${props.getProperty(configKey)}" }
         .mkString(" ")
     },
-    resolvers += Resolver.bintrayRepo("scalacenter", "releases")
+    resolvers += Resolver.bintrayRepo("scalacenter", "releases"),
+    semanticdbEnabled := true,
```

Comment:
Weirdly enough tests started to fail due to a missing semanticdb :thinking: Might it be due to target root not set/read properly?